cleanrl/dqn.py

```diff
@@ -92,11 +96,11 @@ def __init__(self, env):
             nn.Linear(84, env.single_action_space.n),
         )
 
-    def forward(self, x):
+    def forward(self, x: torch.FloatTensor) -> torch.FloatTensor:
```

Comment:
I've used FloatTensor here but we should just use `torch.Tensor` if type hints are pursued further.